lib/matplotlib/axes/_axes.py

```diff
@@ -6331,7 +6339,7 @@ def pcolorfast(self, *args, alpha=None, norm=None, cmap=None, vmin=None,
 
         if vmin is not None or vmax is not None:
             ret.set_clim(vmin, vmax)
-        else:
+        elif np.ndim(C) == 2:
             ret.autoscale_None()
```

Comment:
I would add a comment
~~~
# np.dim(C) == 3 has explicit RGB(A) values and does not need scaling.
~~~

Reply:
sure, done

Reply:
Have you pushed? I don't see a comment.

Reply:
duh, put it at the wrong line; fixed.